### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by edent**
As per discussion - add the dependencies.

### CODE REVIEW

**Comment by soreau** [comment]
In your commit message you could put "Closes #19" so that #19 is closed if this is committed.

The problem I see here, is that it doesn't say anything about meson or ninja.

**Comment by edent** [comment]
It does have `apt install meson` in it. I can add in ninja as well, although I didn't seem to have to install that manually.

**Comment by soreau** [comment]
Ah yes, it does list meson, but ninja isn't in repos.. maybe you could make the list without the `sudo apt install` prefix and let folks figure it out for themselves. We can't really say how to install the dependencies on every distribution.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ For Wayfire and wf-shell, default configuration files will also be installed to
 
 The script also has a few other options, which you can see by calling `./install.sh --help`
 
+## Dependencies
+
+You will need the following dependencies installed:
```
Comment (ammen99) [inline README.md]:
```suggestion
The following is a list of dependencies needed on Ubuntu, similar lists are required on other distributions.
```

**Review by ammen99 — APPROVED** [review]
Thanks!

**Pull request merged**